src/Components/Archive/ArchivePage.jsx

```diff
@@ -1,7 +1,52 @@
 import React from 'react';
+import { useArchive, useNotes, useTrash } from '../../context/';
 
 const ArchivePage = () => {
-	return <div>ArchivePage</div>;
+	const { archiveNotes, deleteArchiveNotes, archiveToNotes } = useArchive();
```

Comment:
Suggestion: You can improve the naming of archiveToNotes to something like sendArchiveToNotes as it becomes immediately apparent that it is a function sending data from archive to notes.
 I try to incorporate an action verb in my function name so that it becomes instantly recognizable.
 Like: getSomething, doSomething etc.

Reply:
done